src/helpers/fallbackauthdialog.js

```diff
@@ -4,13 +4,38 @@
  * This file shows how to use MSAL.js to get an access token to Microsoft Graph an pass it to the task pane.
  */
 
-/* global console, localStorage, Office, require */
+/* global console, localStorage, Office */
 
-const Msal = require("msal");
+import { PublicClientApplication } from "@azure/msal-browser";
+
+const msalConfig = {
+  auth: {
+    clientId: "{application GUID here}", //This is your client ID
+    authority: "https://login.microsoftonline.com/common",
+    redirectUri: "https://localhost:{PORT}/fallbackauthdialog.html",
+    navigateToLoginRequestUrl: false,
+  },
+  cache: {
+    cacheLocation: "localStorage", // Needed to avoid "User login is required" error.
+    storeAuthStateInCookie: true, // Recommended to avoid certain IE/Edge issues.
+  },
+};
+
+var loginRequest = {
+  scopes: [`https://graph.microsoft.com/User.Read`],
```

Comment:
The client should not request any Graph scopes. Instead use
```javascript
const loginRequest = {
"scopes": [accessScope],
"extraScopesToConsent": ["user.read"]
};
```

Which will get the consent but not add the scope to the access token. See [this Node JS sample code](https://github.com/OfficeDev/Office-Add-in-samples/blob/2a70eed92aa66112f907577863f8b779210e12f0/Samples/auth/Office-Add-in-NodeJS-SSO/Complete/public/javascripts/fallback-msal/authConfig.js#L14) for more details.

Reply:
That also requires the 'const accessScope = "api://" + window.location.host + "/" + clientId + "/access_as_user";' line in code.  Would this access scope be the same in my case as is it for the sample?

Reply:
Made the modification.